lms/djangoapps/static_template_view/views.py

```diff
@@ -47,6 +47,13 @@ def render(request, template):
 
     url(r'^jobs$', 'static_template_view.views.render', {'template': 'jobs.html'}, name="jobs")
     """
+    redirect_urls = {
```

Comment:
This is not the right way, it will affect openedx instances as well. You should add marketing URL mapping here https://github.com/edx/edge-internal/blob/d3b65b770200cbacbb255a23ab5efd61eff06be2/edge-remote-config/prod/lms.yml#L762.